src/utils/src/validate-wbs.ts

```diff
@@ -11,13 +11,11 @@ import { WbsNumber } from './types/project-types';
  * @param wbsNum WBS number to validate
  */
 export const validateWBS = (wbsNum: string): WbsNumber => {
-  const errorMsg: string = 'WBS Invalid: ';
+  const errorMsg: string = `WBS "${wbsNum}" is Invalid: `;
   if (wbsNum == null || wbsNum === undefined) {
     throw new Error(errorMsg + 'given WBS # is null');
   }
-  if (wbsNum.match(/\./g) == null) {
```

Comment:
I'm not convinced this is a good thing to remove, and it would also require updating tests